File: TYPESCRIPT/Cap04/c04ex10.ts
// c04ex10.ts

import * as rls from 'readline-sync';

process.stdout.write("Entre valor para a variavel <A>: ");
const A: number = parseInt(rls.question());

process.stdout.write("Entre valor para a variavel <B>: ");
const B: number = parseInt(rls.question());

process.stdout.write("Entre valor para a variavel <X>: ");
const X: number = parseInt(rls.question());

let C: number;
if (!(X > 5))
  C = A + B;
else
  C = A - B;

console.log("");
console.log("Resultado = " + C);

console.log("");
process.stdout.write('Tecle <Enter> para encerrar... ');
rls.question();

File: JULIA/Cap04/c04ex10.jl
# c04ex10.jl

print("Entre valor para a variavel <A>: ")
a = parse(Int, readline())

print("Entre valor para a variavel <B>: ")
b = parse(Int, readline())

print("Entre valor para a variavel <X>: ")
x = parse(Int, readline())

if (!(x > 5))
  c = a + b
else
  c = a - b
end

println()
println("Resultado = $c")

println()
print("Tecle <Enter> para encerrar... ")
readline()
